_docs/kustomize/akashd/service.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: akash
 spec:
   selector:
-    app: akash
+    app: akashd
```

Comment:
should it be `akash` ?

Reply:
I think we can name it whatever we like here. The other `service.yaml` for `akash-provider` is named `akash-provider` so I am trying to be consistent with names.

Reply:
I believe this is correct, it's the Selector, so it needs to match the `akashd` deployments metadata label, which is `app: akashd`. So it was wrong before :grimacing:

Reply:
oh, make sense